.github/workflows/build-docker.yml

```diff
@@ -22,9 +22,9 @@ jobs:
       fail-fast: false
       matrix:
         special: [null]
-        geometry: ["orange", "vecgeom"]
+        geometry: ["vecgeom"]
```

Comment:
Reminder: revert.

Reply:
Yep when I do the "revertme " I delete the "all" job so it's impossible to merge till we restore it :)